app/preload/database/addressBookDao.ts

```diff
@@ -47,6 +47,18 @@ async function findEntries(
         .then((e) => e.map(sanitizeAddressBookEntry));
 }
 
+export async function getEntryName(address: string) {
+    const entry = await (await knex())
+        .select<AddressBookEntry>()
```

Comment:
You should be able to select the name directly in the SQL query:
```
.select('name')
```

Reply:
Yes.